pkg/logging/json_logger.go

```diff
@@ -230,3 +242,21 @@ func viewLogsJSONFileThroughTailExec(lvopts LogViewOptions, jsonLogFilePath stri
 
 	return jsonfile.Decode(stdout, stderr, r, lvopts.Timestamps, lvopts.Since, lvopts.Until, 0)
 }
+
+func filterTailStderr(reader io.Reader) error {
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasSuffix(line, "has appeared;  following new file") ||
+			strings.HasSuffix(line, "has become inaccessible: No such file or directory") ||
+			strings.HasSuffix(line, "has been replaced;  following new file") {
+			continue
```

Comment:
Sorry, could you please explain these suffixes and why to ignore them, thanks

Reply:
HI @fahedouch 

If the code not exist, the `nerdctl logs` will print:
```
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |tail: '/var/lib/nerdctl/1935db59/containers/k8s.io/5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac/5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac-json.log' has become inaccessible: No such file or directory
wordpress-1 |tail: '/var/lib/nerdctl/1935db59/containers/k8s.io/5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac/5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac-json.log' has appeared;  following new file
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |tail: '/var/lib/nerdctl/1935db59/containers/k8s.io/5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac/5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac-json.log' has become inaccessible: No such file or directory
wordpress-1 |tail: '/var/lib/nerdctl/1935db59/containers/k8s.io/5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac/5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac-json.log' has appeared;  following new file
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
wordpress-1 |1712751906
```

The behavior is not compatible with docker.

So, using a function called `filterTailStderr` to filter the known error,

Reply:
what happens if you make it quite wit `-q` flag ?

Reply:
`tail -q -F` is the same with `tail -F`:
```
# tail -F -q 5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac-json.log
{"log":"1712798645\n","stream":"stdout","time":"2024-04-11T01:24:05.959623997Z"}
{"log":"1712798645\n","stream":"stdout","time":"2024-04-11T01:24:05.960589592Z"}
{"log":"1712798645\n","stream":"stdout","time":"2024-04-11T01:24:05.961543894Z"}
tail: '5e847577558e46eecff13b4b4096d65dcade215105673c56c5a1ae3ab80bd9ac-json.log' has been replaced;  following new file
{"log":"1712798645\n","stream":"stdout","time":"2024-04-11T01:24:05.96254524Z"}
{"log":"1712798645\n","stream":"stdout","time":"2024-04-11T01:24:05.963993314Z"}
{"log":"1712798645\n","stream":"stdout","time":"2024-04-11T01:24:05.964718232Z"}
{"log":"1712798645\n","stream":"stdout","time":"2024-04-11T01:24:05.96635917Z"}
```